### ISSUE TITLE
[unicorn/import-style] false positive for importing some TypeScript interfaces

### ISSUE THREAD

**Issue by leonsilicon** [sindresorhus/eslint-plugin-unicorn#1795]
When importing the TypeScript interface `ChalkInstance` from `chalk`:

```typescript
import { ChalkInstance } from "chalk";
```

`eslint-plugin-unicorn` gives the following error:

```
/Users/leonzalion/projects/playground/chalk/test.ts
  1:1  error  Use default import for module `chalk`  unicorn/import-style

✖ 1 problem (1 error, 0 warnings)
```

However, there is no default import for the `ChalkInstance` type.

My guess is that it has something to do with the following lines in the chalk type definitions:

```typescript
declare const chalk: ChalkInstance;
export default chalk;
```

**Repro:** https://github.com/leonzalion/unicorn-import-style-bug

**Comment by alecmev** [comment]
A `type` modifier - `import { type ChalkInstance } from "chalk"` - doesn't help either, unfortunately.
